test/verify/check-docker-storage

```diff
@@ -210,6 +165,13 @@ class TestDockerStorageMapper(MachineCase):
 
         # Add a disk
 
+        # HACK - The total size should increase by 100 MiB, but it
+        # doesn't always because docker-storage-setup needs to be
+        # specifically told to grow the root filesystem.  So we just
+        # print some numbers for now to see what it's happening.
+
+        print("Initially", self.total_size())
```

Comment:
Presumably debugging leftover, but I'm actually in favor of leaving that in.

Reply:
Yes, debugging is on-going. :-)